setup.py

```diff
@@ -12,13 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 from setuptools import setup, find_packages
 import pathlib
 
+from distutils.util import strtobool
+
 here = pathlib.Path(__file__).parent.resolve()
 
 long_description = (here / "README.md").read_text(encoding="utf-8")
 
+RAPIDS_VERSION = "24.10.0a0" if strtobool(os.getenv("IS_NIGHTLY", "False")) else "24.8"
```

Comment:
I would suggest pinning to `==24.8.*` for stable builds and `>=24.10.0a0,<=24.10` for nightly builds. That's the recommendation of the [RAPIDS install guide](https://docs.rapids.ai/install), as well.